data/args.yml

```diff
@@ -2,7 +2,7 @@
 version: "1.18"
 
 # The full Istio version identifier the docs describe
-full_version: "1.18.2"
+full_version: "1.18.3"
```

Comment:
This change should not be here. Noticed you are putting this against the `release-1.18` branch, but it should be `master`.